ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -891,21 +916,32 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
             <div style={{...styles.header, color: colors.primary, fontSize: 14, marginBottom: '0.2rem'}}>
               Select account
             </div>
-            <Dropdown style={{width: '14rem'}}
-                      value={this.state.workspace.billingAccountName}
-                      options={this.buildBillingAccountOptions()}
-                      onChange={e => {
-                        if (e.value === CREATE_BILLING_ACCOUNT_OPTION_VALUE) {
-                          this.setState({
-                            showCreateBillingAccountModal: true
-                          });
-                        } else {
-                          this.setState(fp.set(['workspace', 'billingAccountName'], e.value));
-                        }
-                      }}
-            />
-          </WorkspaceEditSection>
-        }
+            <FlexRow>
```

Comment:
I believe the flex has done strange things to the alignment of the dropdown selection. If you look, it is aligned to the top of the container rather than the center. Could you fix the styling on that? Should be either just putting an alignItems and maybe a fixed height on the dropdown?
New:
<img width="345" alt="Screen Shot 2020-04-10 at 2 39 44 PM" src="https://user-images.githubusercontent.com/15351021/79014561-273ec980-7b39-11ea-9b62-32351d4325c8.png">
Old:
<img width="349" alt="Screen Shot 2020-04-10 at 2 39 49 PM" src="https://user-images.githubusercontent.com/15351021/79014580-2dcd4100-7b39-11ea-986d-87cb7cde947c.png">

Reply:
Subtle!

I was able to keep the FlexRow by removing some padding from the Clickable/div I added to the row